presto-native-execution/presto_cpp/main/functions/README.md

```diff
@@ -0,0 +1,23 @@
+# Prestissimo: Dynamically Loading Function Library
+
+This library adds the ability to load User Defined Functions (UDFs) without having to fork and build Prestissimo through the use of shared libraries. The Prestissimo worker is to access said code. The dynamic functions are to be loaded upon running an instance of the presto server. In the presto server instance, it will search for any .so or .dylib files and load them using this library.
+
+## Getting started
+
+The Process can be roughly broken down into the steps below:
+### 1. Create a cpp file for your dynamic library
+For dynamically loaded function registration, the format followed is mirrored of that of built-in function registration with some noted differences. For instance, the below example function uses the extern "C" keyword to protect against name mangling. Additionally, a registry() function call is also necessary here.
```

Comment:
@steveburnett I was thinking here, for clarity, I could copy paste the test function in MyDynamicTestFunction.cpp so it clearly demonstrates the format of the function. do you think that would be a useful addition or repetitive as the reader can find the test function for themselves too

Reply:
I think it might be repetitive, but copy-pasting the test function in MyDynamicTestFunction.cpp here could be valuable. It would depend how you write it. I could see it useful if you want to emphasize certain details, and if you keep the copied portion to the minimum number of lines needed so it doesn't overwhelm your steps here in the README. 

I'd suggest that you try it and decide for yourself if you think it works or not. 

If the part you need to copypaste is large, maybe discuss the test function here and include a link to the file MyDynamicTestFunction.cpp for readers.

Reply:
Nice! I think the discussion you provide works, and the user now has a link to the example code in MyDynamicTestFunction.cpp if they want it.